### DISCUSSION TITLE
[Feature Request] 账号密码登录

### DISCUSSION THREAD

**Discussion by OXeu** [OXeu/Rin#62] [Ideas]
**您的功能请求是否与问题相关？请描述**

接入 Github OAuth 虽然比较简单，但是如果原生支持账号密码登录将会更加方便部署，同时能够解决部分地区访问 GitHub 缓慢导致难以登录的问题

**描述您想要的解决方案**

支持账号密码登录

**描述你考虑过的替代方案**

无

**其他信息**

同时需要考虑是否增加免登录评论的支持（需要输入一些必须信息）

**Comment by flyingcrp** [comment]
请问这个有什么进展吗?

**Reply by OXeu** [reply]
目前主要在开发动态配置相关的内容，待其完成后会推进账号密码登录
